test/features/admin_reporting.feature

```diff
@@ -38,4 +38,36 @@ Feature: AdminReporting
         And I click the link with text that contains "Admin Report"
         And I press the element with xpath "//button[contains(string(), 'Show')]"
         Then I should see an element with xpath "//tr[@id='overdue-datasets']/td[contains(@class, 'metric-title') and contains(string(), 'Overdue Datasets') and position()=1]"
-        Then I should see an element with xpath "//tr[@id='overdue-datasets']/td[contains(@class, 'metric-data') and position()=2]"
+        And I should see an element with xpath "//tr[@id='overdue-datasets']/td[contains(@class, 'metric-data') and position()=2]"
+
+    Scenario: As an admin user of my organisation, when I view my admin report, I can verify that datasets without groups are identified
```

Comment:
Why do we include reports in harvester plugin?

Reply:
Regression testing, ensuring that we haven't busted some aspect of `ckanext-data-qld`.